packages/core/src/Tooltip/index.tsx

```diff
@@ -96,7 +96,7 @@ const ExpandedTooltipTitle = styled(Typography).attrs({
   variant: 'chip-tag-text',
 })`
   font-weight: ${font.fontWeight.semibold};
-  white-space: nowrap;
+  white-space: normal;
```

Comment:
I think "normal" is the default value. Does it work with just removing this line?

Reply:
Normal is the default yes, so removing it would have the same effect.

Reply:
Fixed